sports/views.py

```diff
@@ -16,7 +19,56 @@ class ParticipantDetail(DetailView):
 class ParticipantCreate(CreateView):
     model = Participant
     form_class = ParticipantForm
-    template_name = 'add_form.html'
+    template_name = 'multiple_add_forms.html'
+    success_url = reverse_lazy('participants')
+    performance_formset = modelformset_factory(Performance, form=PerformanceForm, extra=0)
+
+    def get(self, request, *args, **kwargs):
+        """
+        Handles GET requests and instantiates blank versions of the form
+        and its inline formsets.
+        """
+        self.object = None
+        form_class = self.get_form_class()
+        participant_form = self.get_form(form_class)
+        participant_form.title = _('Participant')
+        performance_form = self.performance_formset()
+        performance_form.title = _('Performance')
+        forms = [participant_form, performance_form, ]
+        return self.render_to_response(
+            self.get_context_data(forms=forms)
+        )
+
+    def post(self, request, *args, **kwargs):
+        """
+        Handles POST requests, instantiating a form instance and its inline
+        formsets with the passed POST variables and then checking them for
+        validity.
+        """
+        self.object = None
```

Comment:
Because we're basically combining 2 forms into one (a participant form and a performance form) but one needs to be checked before the other can be filled in.